docs/clients/adc_interface.rst

```diff
@@ -0,0 +1,115 @@
+ADC Interface
+-------------
+Vertiq's :ref:`ADC Interface <manual_adc_interface>` provides access to an on-board Analog to Digital Converter (ADC). An ADC makes it possible for your module to read input analog voltages. The ADC handles voltages from 0.0V to 3.3V with a 12-bit resolution. For example, if you input 1V to the ADC interface, reading the voltage would return 1V, and reading the "raw value" would return 1241 (:math:`\frac{V_{\text{in}} * 4096}{3.3}`). 
+
+The ADC interface provides read-only access to both the voltage read and the raw ADC value.
+
+Arduino
+~~~~~~~
+
+To use the ADC Interface in Arduino, ensure adc_interface_client.hpp is included. This
+allows the creation of a AdcInterfaceClient object. See the Message Table below for available messages. 
+All message objects use the Short Name with a trailing underscore. 
+All messages use the standard Get/Set/Save functions. 
+
+A minimal working example for the AdcInterfaceClient is:
```

Comment:
italics around *AdcInterfaceClient*

Reply:
I don't believe any other client files have italics. Should I change it for all of them?